### PR TITLE
docs: update headless and stencil getting started guide into separate…

### PR DESCRIPTION
**Pull request by bc-micah**
Jira: [DEVDOCS-5858](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5858)

## What/Why?
- Separate the headless and stencil guides to make it clearer and easier to understand.

## Rollout/Rollback
Revert

## Testing
N/A - Docs

[DEVDOCS-5858]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5858?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bc-victor — CHANGES_REQUESTED** [review]

File: docs/headless.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+- Script for the `<body>` tag and make sure to replace the `{REPLACE_WITH_YOUR_STORE_HASH}` and `{REPLACE_WITH_YOUR_CHANNEL_ID}` with the appropriate information
+
+```html
+<script type="module" src="http://localhost:3001/src/main.ts"></script>
+<script>
+  window.B3 = {
+    setting: {
+      store_hash: '{REPLACE_WITH_YOUR_STORE_HASH}',
+      channel_id: { REPLACE_WITH_YOUR_CHANNEL_ID },
```
Comment (bc-marco) [inline docs/headless.md:31]: I think curly braces can cause confusion, so it should be better remove them
```suggestion
- Script for the `<body>` tag and make sure to replace the `REPLACE_WITH_YOUR_STORE_HASH` and `REPLACE_WITH_YOUR_CHANNEL_ID` with the appropriate information

```html
<script type="module" src="http://localhost:3001/src/main.ts"></script>
<script>
  window.B3 = {
    setting: {
      store_hash: 'REPLACE_WITH_YOUR_STORE_HASH',
      channel_id: REPLACE_WITH_YOUR_CHANNEL_ID,
```

File: docs/headless.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  RefreshRuntime.injectIntoGlobalHook(window)
+  window.$RefreshReg$ = () => {}
+  window.$RefreshSig$ = () => (type) => type
+  window.__vite_plugin_react_preamble_installed__ = true
+</script>
+<script type="module" src="http://localhost:3001/@vite/client"></script>
+```
+
+- Script for the `<body>` tag and make sure to replace the `{REPLACE_WITH_YOUR_STORE_HASH}` and `{REPLACE_WITH_YOUR_CHANNEL_ID}` with the appropriate information
+
```
Comment (bc-marco) [inline docs/headless.md:24]: Take advantage of markdown util
```suggestion
> [! WARNING]  
> Platform is required to load buyer portal properly, select one depending on [your store channel platform](https://developer.bigcommerce.com/docs/rest-management/channels#platform)
```

File: docs/headless.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+<script>
+  window.b3CheckoutConfig = {
+    routes: {
+      dashboard: '/account.php?action=order_status',
+    },
+  }
+  window.B3 = {
+    setting: {
+      store_hash: '<YOUR_STORE_HASH>',
+      channel_id: '<YOUR_CHANNEL_ID>',
```
Comment (bc-marco) [inline docs/headless.md:74]: For consistency change this to
```suggestion
      store_hash: 'REPLACE_WITH_YOUR_STORE_HASH',
      channel_id: REPLACE_WITH_YOUR_CHANNEL_ID,
```

File: docs/headless.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  nomodule=""
+  crossorigin=""
+  src="<YOUR_APP_URL_HERE>/index-legacy.*.js"
+></script>
+```
+
+Replace the following values:
+- `<VITE_ASSETS_ABSOLUTE_PATH>` with the value you used for the environment variable `VITE_ASSETS_ABSOLUTE_PATH` (where your build is hosted)
+- `<YOUR_STORE_HASH>` with your store hash
+- `<YOUR_CHANNEL_ID>` with the headless channel id
```
Comment (bc-marco) [inline docs/headless.md:108]: Also here
```suggestion
- `REPLACE_WITH_YOUR_STORE_HASH ` with your store hash
- `REPLACE_WITH_YOUR_CHANNEL_ID ` with the headless channel id
```

File: docs/stencil.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+<script>
+  window.b3CheckoutConfig = {
+    routes: {
+      dashboard: '/account.php?action=order_status',
+    },
+  }
+  window.B3 = {
+    setting: {
+      store_hash: '<YOUR_STORE_HASH>',
+      channel_id: '<YOUR_CHANNEL_ID>',
```
Comment (bc-marco) [inline docs/stencil.md:94]: As this is for stencil, it's okay to use
```suggestion
      store_hash: '{{settings.store_hash}}',
      channel_id: {{settings.channel_id}},
```

File: docs/stencil.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+  nomodule=""
+  crossorigin=""
+  src="<YOUR_APP_URL_HERE>/index-legacy.*.js"
+></script>
+```
+
+Replace the following values:
+- `<VITE_ASSETS_ABSOLUTE_PATH>` with the value you used for the environment variable `VITE_ASSETS_ABSOLUTE_PATH` (where your build is hosted)
+- `<YOUR_STORE_HASH>` with your store hash
+- `<YOUR_CHANNEL_ID>` with the headless channel id
```
Comment (bc-marco) [inline docs/stencil.md:128]: This would not be required
```suggestion
```

**Review by bc-marco — CHANGES_REQUESTED** [review]
Please check the comments below

**Review by bc-marco — APPROVED** [review]

**Review by bc-victor — APPROVED** [review]

**Pull request merged**